buzsaki_lab_to_nwb/huszar_hippocampus_dynamics/behaviorinterface.py

```diff
@@ -106,8 +154,8 @@ def run_conversion(self, nwbfile: NWBFile, metadata: dict, stub_test: bool = Fal
         self.session_path = Path(self.source_data["folder_path"])
         self.session_id = self.session_path.stem
 
-        module_name = "Figure - 8 maze"
-        module_description = "Figure 8 - maze"
+        module_name = "Figure 8 maze"
```

Comment:
Couple things here

1) Is 'figure eight' the way they refer to the maze in the publication?

2) I'd remove spaces from object names (including module objects); these end up as keys in the final `NWBFile` dictionary structure, where it's typically an undesired practice to include spaces, slashes, special characters, etc. They also usually have an attribute way of referencing which really makes spaces problematic

3) I also have never included numerics in module names or descriptions unless they corresponded to actual numeric data values; can you use the full word 'eight' in all references?